frontend/src/app/lib/router.js

```diff
@@ -1,11 +1,6 @@
 /* global ga */
```

Comment:
Nit: This could go away since the GA code moved.

Also, now that this is just a clean router component, could it move from `app/lib` to `app/components` and be less of a special snowflake?